.ci/azure/windows.yml

```diff
@@ -142,7 +142,7 @@ jobs:
       python -m pip install -r $(REPO_DIR)\src\bindings\python\wheel\requirements-dev.txt
       python -m pip install -r $(REPO_DIR)\src\bindings\python\requirements.txt
       rem For running Paddle frontend unit tests
-      python -m pip install -r $(REPO_DIR)\src\frontends\paddle\tests\requirements.txt
+      #python -m pip install -r $(REPO_DIR)\src\frontends\paddle\tests\requirements.txt
```

Comment:
let's add comment to return it back once PDPD 2.5 is out

Reply:
Done.